README.md

```diff
@@ -237,6 +237,20 @@ In frontend directory, you will see that `.beaker` is in here. It is actually sy
 
 ---
 
+### Customization
+
+If you would like to use another contract template, feel free to change the configuration
+
+```toml
+# In Beaker.toml, change the template_repo to whatever you want.
+[wasm]
+template_repo = "https://github.com/osmosis-labs/osmosis-bindings.git"
+```
+
+### Specified flag commands
+
+Some useful flag commands are located below. For more, please see [further documentation](https://github.com/xBalbinus/beaker/tree/main/docs/commands) 
```

Comment:
the link should be for the main repo rather than the forked one.

Reply:
Oh lol oops! My mistake, will fix.